core/src/main/scala/cats/data/Ior.scala

```diff
@@ -142,6 +144,11 @@ sealed abstract class IorInstances extends IorInstances0 {
     def pure[B](b: B): A Ior B = Ior.right(b)
     def flatMap[B, C](fa: A Ior B)(f: B => A Ior C): A Ior C = fa.flatMap(f)
   }
+
+  implicit def bifunctor: Bifunctor[Ior] =
+    new Bifunctor[Ior] {
+      override def bimap[A, B, C, D](fab: A Ior B)(f: (A) => C, g: (B) => D): C Ior D = fab.bimap(f, g)
```

Comment:
Could you also remove the parens from `f` and `g`? If functions have only one parameter we usually omit them.